tools/scripts/create_universal_linux_dds_idl_binaries.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+###########################################################################################################################
+# This script is used to create dds and tao IDL binaries without the dependencies on libstdc++ shared libraries;
+# however, they still depend on libc.so and
```

Comment:
This looks unfinished...

also please document which systems this script can run on, and what system packages (or other software) needs to be installed